core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -115,7 +115,7 @@ public boolean getRequiresConfirmation() {
      * @return permission required for user to access this management link, in addition to {@link Jenkins#ADMINISTER}
```

Comment:
```suggestion
     * @return permission required for user to access this management link; by default, {@link Jenkins#ADMINISTER}
```

?

Reply:
More information about the changes in this method can be found in https://github.com/jenkinsci/jep/pull/269

Reply:
I do not see anything in a quick reading of that PR that addresses my question and the one below:

* If `ADMINISTER` rather than `null` is now the default, under what conditions would a plugin be overriding this to return null, and why should we even permit that?
* If the returned permission is “in addition to” `ADMINISTER`, then what sense would it make to return `MANAGE`, since `ADMINISTER` + `MANAGE` = `ADMINISTER` anyway?

Am I missing a particular subtle point in that PR that should be called out here?

Reply:
https://github.com/jenkinsci/jenkins/pull/4501/files#diff-841df66889e3c3214c68de134dfe05beR1065  seems to show that ManagementLinks are checking this and only this permission.

Reply:
Not sure I understand. If others do understand and the code is correct as written (null is allowed, and this is actually in addition to `ADMINISTER`) then I guess ignore me (or improve documentation).

Reply:
Nothing I have seen in the code makes me believe that this was ever implemented to allow for "in addition to `ADMINISTER`", and my best guess is that allowing `null` was just a shortcut to allow implementations to be accessible regardless of what permission the user has:

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/Functions.java#L874

I can not think of an instance where it makes sense for a plugin to override this and return `null` - if someone is going to take the time to update a plugin, they should be encouraged to pick a suitable permission instead.